src/screens/GroupSetting/hooks/useGroupSetting.js

```diff
@@ -2,28 +2,41 @@ import React from 'react';
 import SimpleToast from 'react-native-simple-toast';
 import {launchImageLibrary} from 'react-native-image-picker';
 
-import StringConstant from '../../../utils/string/StringConstant';
+import {Linking} from 'react-native';
+import {useNavigation} from '@react-navigation/core';
+import {useRecoilState} from 'recoil';
 import {Context} from '../../../context';
 import {getChatName} from '../../../utils/string/StringUtils';
 import {requestExternalStoragePermission} from '../../../utils/permission';
-import {uploadFile} from '../../../service/file';
+import SignedMessageRepo from '../../../service/repo/signedMessageRepo';
+import ChannelList from '../../../database/schema/ChannelListSchema';
+import useLocalDatabaseHook from '../../../database/hooks/useLocalDatabaseHook';
+import {CHANNEL_GROUP} from '../../../hooks/core/constant';
+import StringConstant from '../../../utils/string/StringConstant';
+import {chatAtom} from '../../../hooks/core/chat/useChatUtilsHook';
+import useUploadImage from '../../../hooks/useUploadImage';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new code has added several new imports and removed one (`uploadFile`). The removal of `uploadFile` is justified as the image upload functionality is now handled by the `useUploadImage` hook. However, it's important to ensure that all newly added dependencies are properly installed and imported from the correct paths.

```diff
- import {uploadFile} from '../../../service/file';
+ import {Linking} from 'react-native';
+ import {useNavigation} from '@react-navigation/core';
+ import {useRecoilState} from 'recoil';
+ import SignedMessageRepo from '../../../service/repo/signedMessageRepo';
+ import ChannelList from '../../../database/schema/ChannelListSchema';
+ import useLocalDatabaseHook from '../../../database/hooks/useLocalDatabaseHook';
+ import {CHANNEL_GROUP} from '../../../hooks/core/constant';
+ import {chatAtom} from '../../../hooks/core/chat/useChatUtilsHook';
+ import useUploadImage from '../../../hooks/useUploadImage';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->